aiida/cmdline/params/options/commands/setup.py

```diff
@@ -152,6 +152,18 @@ def get_quicksetup_password(ctx, param, value):  # pylint: disable=unused-argume
     return value
 
 
+SETUP_PROFILE_UUID = options.OverridableOption(
+    '--profile-uuid',
+    prompt='Profile UUID',
+    prompt_fn=lambda x: False,  # This option should not be prompted for.
+    help='The UUID of the profile. This should only be defined if configuring a profile that should connect to the '
+    'same services of a profile that is already configured in another AiiDA instance, in which case the profile UUIDs '
+    'need to be identical.',
+    required=False,
+    type=str,
```

Comment:
should validation be added here?

some code might rely on the profile uuid actually being a uuid

Reply:
I thought about this, but the API actually also doesn't have any validation. Any string will be accepted and will also work. Well, I am not sure if _any_ string would work, since it is used for the RabbitMQ resources and that might put restrictions on it.

It would maybe be good to add validation on the `Profile` class on what is acceptable for the `profile_uuid` key. But until that isn't there, I don't think we should put validation on the CLI. I can add this in another PR.

Reply:
@ltalirz you ok with merging this as is? I can open an issue to address the validation of the profile UUID more generally

Reply:
yep, go ahead